.hashibot.hcl

```
@@ -584,7 +585,9 @@ behavior "pull_request_path_labeler" "service_labels" {
     ]
     "service/apigatewayv2" = [
       "**/*_api_gateway_v2_*",
+      "**/*_apigatewayv2_*",
       "**/api_gateway_v2_*"
```

Not sure how sensitive hashibot's HCL is for the missing mid-list comma here:

```suggestion
      "**/api_gateway_v2_*",
```